javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,46 @@
+
+//Flight booking fullname function
+
+function getFullName(firstName, surname, useFormalName = true, gender = "") {
+    if (useFormalName) {
+        if (gender === "male") {
+            return "lord " + firstName + surname;
+        } else if (gender === "female") {
+            return "lady " + firstName + surname;
+        }
+    }
+            return firstName + surname;
+        }
+    
+
+    console.log(getFullName("Benjamin ", "Hughes", true,"male"))
+    console.log(getFullName("Benjamin ", "Hughes", false))
+    console.log(getFullName("Stefany"," Romero",true,"female"))
+
+
+//Event Application
+
+function getEventWeekday(daysFromToday){
+
+const weekDays= [,"Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
+const today = new Date().getDate();
+const eventDay = (today + daysFromToday) % 7;
+return weekDays[eventDay];
+
+}
+console.log(getEventWeekday(9));
+console.log(getEventWeekday(2)); 
+
+//Weather wear
+
+function clothesTowear(temperature){
```

Comment:
it seems that there is a very small mistake in the writing of the name of the function.
`clothesToWear` just a capital W actually ensures a proper usage of the camel case.
it is a small thing, but just to make better readability.